web/components/bet/sell-panel.tsx

```diff
@@ -146,32 +151,60 @@ export function SellPanel(props: {
       }
     }
   }
+  const onStonkAmountChange = (amount: number | undefined) => {
+    setAmountValue(amount)
+    if (totalSaleValue - (amount ?? 0) < 1) setAmount(shares)
+    else setAmount(((amount ?? 0) / totalSaleValue) * shares)
```

Comment:
It also doesn't make sense to change the sell panel just for stocks either. We should port all markets to use sale value instead of shares eventually.

Reply:
Dang, I think stonks would be much better to use sale value with...